extention/app.js

```
@@ -0,0 +1,217 @@
+function terminate() {
+	// The `click` listener is automatically removed after it has been called once
+	window.removeEventListener('mousemove', throttle(updateHighlight));
+	window.removeEventListener('keydown', checkTerminateKeys);
+	removeHighlight();
+};
+
+
+
+function createStickyDiv() {
+    // Create the container div
+	debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)